eden/scm/setup.py

```diff
@@ -1669,6 +1669,12 @@ def cythonize(*args, **kwargs):
                 "depends": [versionhashpath],
                 "include_dirs": ["contrib/chg"] + include_dirs,
                 "extra_args": filter(None, cflags + chgcflags + [STDC99, WALL, PIC]),
+                # chg uses libc::unistd/getgroups() to check that chg and the
+                # sl cli have the same permissions (see 49ee17f2bfad).
```

Comment:
Nit: I'd write `D43676809` instead of the open source Git commit hash (`49ee17f2bfad`).

Reply:
off-topic: sapling doesn't support referencing commits by Diff Revision:

```sh
❯ sl show D43676809
abort: unknown revision 'D43676809'!
```

that'd be a neat feature to add, but it'd only make sense in open source FB repos

Reply:
> it'd only make sense in open source FB repos

[LLVM too!](https://github.com/llvm/llvm-project/commits/main)

Reply:
forgot to amend my change.  oops c: